### ISSUE TITLE
in the photos are a female and a male of Caliadurgus fasciatellus in copula

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5338]
**in the photos are a female and a male of Caliadurgus fasciatellus in copula**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/602f0e1458ba3b9ab4275c48108a1cb4:7a836dda4aab9340cdf9cb1101d9ca501484a330289b5e7de8d2872b2f9b7a5837e5641ef8cb873a3b47259ac50a3db7307e3f47818f3e9b1a956c3ada34e115c189fcae0b57570dc344a8d491f6ca56075a5efa9ee1b136836138271f7fbb4b5e2d137164a23143a6cb067ba7d5fb80) - [Send email](https://www.gbif.org/api/feedback/user/mailto/602f0e1458ba3b9ab4275c48108a1cb4:7a836dda4aab9340cdf9cb1101d9ca501484a330289b5e7de8d2872b2f9b7a5837e5641ef8cb873a3b47259ac50a3db7307e3f47818f3e9b1a956c3ada34e115c189fcae0b57570dc344a8d491f6ca56075a5efa9ee1b136836138271f7fbb4b5e2d137164a23143a6cb067ba7d5fb80)
System: Chrome 125.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/uk/occurrence/2329560169
Window size: width 1536 - height 703
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-05-24T21:00:53.365Z',to:'2024-05-24T21:06:53.365Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-05-24T21:00:53.365Z',to:'2024-05-24T21:06:53.365Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: INFO
datasetKey: 6ac3f774-d9fb-4796-b3e9-92bf6c81c084
publishingOrgKey: bb646dff-a905-4403-a49b-6d378c2cf0d9

Node handles: @jholetschek

**Comment by jhnwllr** [comment]
@jholetschek

https://nabu-naturgucker.de/natur.dll/pXSXFhuG0o5gbO9fT5nmceyxu9q/

Appears to be correct at source, so it will get updated corrected on next ingestion probably.

**Comment by CecSve** [comment]
@jholetschek it seems that the name is updated in naturgucker (Caliadurgus fasciatellus) but not in the GBIF occurrence (Priocnemis melanosoma Kohl, 1880). I wonder if it is an issue with naturgucker --> DwC-A? I tried synchonizing the installation but no changes appeared, so I suspect that the archive might be out of date?

**Comment by MattBlissett** [comment]
https://www.naturgucker.de/biocase/pywrapper.cgi?dsa=biocase&inventory=1 (the endpoint with &inventory=1 added) shows the archive was last generated on 12 March.

**Comment by jholetschek** [comment]
Yes, automated updates seem to be disabled, I'll contact the providers about it.
I've updated the archive manually and triggered a re-ingest.
